mor/pyfeelpp-mor/feelpp/mor/reducedbasis/reducedbasis.py

```diff
@@ -0,0 +1,895 @@
+"""reducedBasisPetsc :
```

Comment:
you merged pyrb into feature.pbdw ?

Reply:
yes, I needed some of the things that @thomas-saigre did in this branch, and it was in several commits.
The branch is ready to be merge also in develop no ?

Reply:
not yet... there are still issues with the error bound, I'm working on it right now, and the parabolic code is not working at all

Reply:
I think it is fine as long as the code and the tests are fine. even if everything is not ready

Reply:
ready meaning coded

Reply:
I try to commit the fixes this morning, if it is ok ?